src/react-slidy-slider.jsx

```diff
@@ -45,6 +48,8 @@ export default class ReactSlidySlider extends Component {
       doAfterSlide: this.handleAfterSlide,
       // fix if the user try to use a `true` value for infinite
       infinite: props.infinite === true ? 1 : props.infinite,
+      // fix if the user try to use a `true` value for hideTailArrows
+      hideTailArrows: props.hideTailArrows === true ? 1 : props.hideTailArrows,
```

Comment:
I don't get why you ask for a boolean but convert that to 1 if true.

If you look `_getShowArrows` you will see that the data models seems strange.

`showArrows` is bool but `infinite` or `hideTailArrows` are int :\